nuvla/job_engine/connector/nuvlaedge_docker.py

```diff
@@ -763,3 +770,163 @@ def log(self, component: str, since: datetime, lines: int) -> str:
         return container.logs(timestamps=True,
                               tail=lines,
                               since=since.replace(tzinfo=None)).decode()
+
+    @staticmethod
+    def _unsupported_action_job_response(action: str):
+        return {
+            'success': False,
+            'return-code': 0,
+            'message': f"Unsupported action: {action}"
+        }
+
+    @staticmethod
+    def _unsupported_resource_for_action_job_response(resource: str, action: str):
+        return {
+            'success': False,
+            'return-code': 0,
+            'message': f"Unsupported resource {resource} for action {action}"
+        }
```

Comment:
You could use the method `_new_job_response` below _(if you convert these 2 static methods to class methods_).

Reply:
Agree, `_new_job_response` method was added after these to and overlooked it :)